tvb_build/docker/Dockerfile-multiscale-build

```diff
@@ -0,0 +1,193 @@
+#
+# TVB & NEST simulations environment
+#
+# Install Python 3.7, NEST, then tvb-library from Pypi
+# Finally link tvb-scripts and tvb-nest from Github sources
+#
+
+FROM neurodebian:buster
+MAINTAINER lia.domide@codemart.ro
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG=C.UTF-8
+
+RUN apt-get update; apt-get install -y automake libtool build-essential git vim octave \
+                       wget python3.7 libpython3.7 libpython3-dev libncurses5-dev libreadline-dev libgsl0-dev cython3 \
+                       python3.7-dev python3-pip python3-numpy python3-scipy python3-matplotlib python3-nose \
+                       ipython3 python3-venv python-virtualenv python3-pyqt5 python-dev libgsl-dev libncurses-dev \
+                       subversion cmake zip unzip; \
+    apt-get clean
+
+RUN wget -q http://www.mpich.org/static/downloads/3.1.4/mpich-3.1.4.tar.gz \
+    && tar xf mpich-3.1.4.tar.gz \
+    && cd mpich-3.1.4 \
+    && ./configure --disable-fortran --enable-fast=all,O3 --prefix=/usr \
+    && make -j$(nproc) \
+    && make install \
+    && ldconfig \
+    && cd .. \
+    && rm -rf mpich-3.1.4 \
+    && rm mpich-3.1.4.tar.gz
+
+# Create user different than root
+RUN useradd -ms /bin/bash docker
+USER docker
+ARG HOME=/home/docker
+RUN mkdir $HOME/env; mkdir $HOME/packages
+
+# Create a dedicated Python env for working in it
+ENV VENV=$HOME/env/neurosci
+RUN python3 -m venv $VENV && python3 -m venv --system-site-packages $VENV; \
+    $VENV/bin/pip3 install --upgrade pip; \
+    $VENV/bin/pip3 install parameters quantities neo "django<1.9" django-tagging future hgapi gitpython sumatra nixio; \
+    $VENV/bin/pip3 install --upgrade nose ipython
+# django<1.9 is needed for pynest - september 2019 - NEST 2.18
+
+# Install from sources libneurosim (NEST boost dependency)
+WORKDIR $HOME/packages
+RUN git clone --depth 1 https://github.com/INCF/libneurosim.git; \
+    cd libneurosim; ./autogen.sh; \
+    mkdir $VENV/build
+WORKDIR $VENV/build
+RUN mkdir libneurosim; \
+    cd libneurosim; \
+    PYTHON=$VENV/bin/python $HOME/packages/libneurosim/configure --prefix=$VENV; \
+    make; make install; ls $VENV/lib $VENV/include; \
+    ln -s /usr/lib/python3.7/config-x86_64-linux-gnu/libpython3.7.so $VENV/lib/
+
+# Install NEST, latest stable version (July 2019)
+ARG NEST_VER=2.18.0
```

Comment:
please collect all build args at the top and namespace them  (here they're all `NEST_XYZ` but other below like `LAST_SHA_ROOT` should be `TVB_SHA` etc)

Reply:
We remove the docker file.